taxi/views.py

```diff
@@ -87,3 +104,30 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
```

Comment:
success url here?

Reply:
django redirects user to this new created user page by default.

Reply:
It actually redirects because Driver model has `get_absolute_url` method defined :nerd_face: